torch/csrc/distributed/autograd/context/context.cpp

```diff
@@ -193,6 +202,25 @@ std::shared_ptr<rpc::FutureMessage> DistAutogradContext::
   return state->future;
 }
 
+void DistAutogradContext::setGraphTaskException(const std::string& errorMsg) {
+  std::runtime_error err(errorMsg);
```

Comment:
Why are we doing this? Looks like we convert errorMsg to runtim_error then just get the error message back in err.what() ?